ui/src/shared/constants/tableGraph.js

```diff
@@ -1,3 +1,8 @@
+import calculateSize from 'calculate-size'
+import _ from 'lodash'
+
+export const NULL_COLUMN_INDEX = -1
+export const NULL_ROW_INDEX = -1
 export const NULL_ARRAY_INDEX = -1
```

Comment:
hrrm.. I don't know why NULL_COLUMN/ROW_INDEX got added back. I had changed them to NULL_ARRAY_INDEX. is it because you're using column/row somewhere?

Reply:
I think these snuck in during merging